packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3215,6 +3215,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         _lastTextPosition = currentTextPosition;
         renderEditable.setFloatingCursor(point.state, _lastBoundedOffset!, _lastTextPosition!);
       case FloatingCursorDragState.Update:
+        // Stop cursor blinking and making it visible.
+        _stopCursorBlink(resetCharTicks: false);
+        _cursorBlinkOpacityController.value = 1.0;
```

Comment:
It looks like we already call this on `FloatingCursorDragState.Start`, is something else resetting the cursor blink?

Reply:
Yes, the cursor blinking is reset [here](https://github.com/flutter/flutter/blob/5887d6c74248fc6542a49e4a999b09b2aa58ef9c/packages/flutter/lib/src/widgets/editable_text.dart#L4041C39-L4041C39) when the selection changes. I should note this in the code.

Reply:
Thanks for the explanation. I think given these details, we should probably try prevent the blinking cursor from turning itself back on while the selection is changing due to floating cursor. Maybe by updating the logic in `_startOrStopCursorTimerIfNeeded`.

Reply:
I have made changes to the implementation, could you please take another look?